org.mixedrealitytoolkit.spatialmanipulation/BoundsControl/Visuals/SqueezableBoxVisuals.cs

```diff
@@ -194,6 +194,7 @@ private void Awake()
 
             // Compute flatten vector at startup.
             flattenVector = BoundsCalculator.CalculateFlattenVector(transform.lossyScale);
+            Update();
```

Comment:
We shouldn't be calling Update() in Awake().   Update() should only be called directly by the Unity.

What were you trying to initialize here?

Reply:
Maybe instead of Update(), the squeeze handles should be updated in LateUpdate(), to match the bounds control.

Or can we initialize the squeeze box visuals to some initial state here, without executing all of update()...for example, can we do this?


```cs
float initialActiveFocus = 0;
float initialActivation = 0;
float initialShrink = 0;
WritePropertyValues(boundsRenderer, propertyBlock,, initialActiveFocus , initialActivation , initialShrink);
```

Reply:
with force occlusion function, this is no longer needed